google-api-go-generator/gen.go

```diff
@@ -740,7 +755,7 @@ func (a *API) GenerateCode() ([]byte, error) {
 		pn("// NOTE: prepend, so we don't override user-specified scopes.")
 		pn("opts = append([]option.ClientOption{scopesOption}, opts...)")
 	}
-	pn("opts = append(opts, internaloption.WithDefaultEndpoint(basePath))")
+	pn("opts = append(opts, internaloption.WithDefaultEndpoint(basePath), internaloption.WithDefaultMTLSEndpoint(mtlsBasePath))")
```

Comment:
If above suggestion is accepted, have it only add the option if `mtlsBasePath != ""`.

Reply:
done